src/modules/household/components/EditHouseholdMemberTable.tsx

```diff
@@ -109,21 +106,6 @@ const EditHouseholdMemberTable = ({
     });
   }, [currentMembers]);
 
-  // If new members have beed added highlight their relationship field if it's DNC
```

Comment:
This is no longer needed, because it's not permitted to add a household member without specifying a valid RelationshipToHoH.

The existing "highlight" behavior is retained whereby when you **change the HoH**, the RelationshipToHoH field for other members is highlighted, to indicate that you should review those and update them if they are no longer accurate.

Reply:
Cool, I like the new inference behavior. There are still some edge cases with inference that I guess we might refine later. For example, in a household like this:
- A, HoH
- B, Spouse
- C, Other Relative

When changing HoH from A to C, B stays marked as "Spouse" but should probably be inferred to now be "Other Relative." I think this is not in scope for this PR though - looks like that would be a backend change.

Reply:
@martha Good point, it would be possible to make some additional inferences! Currently the highlight is supposed to nudge the user to review those, and I think it's OK to stick to that existing behavior for now.